### PR TITLE
feat(): add MySQL tutorial

### PR DESCRIPTION
**Pull request by gilgda**

### CODE REVIEW

**Comment by gilgda** [comment]
@Lordnoname  in work

**Comment by Lordnoname** [comment]
@gilgda Please change the usernames in picture 3 and 4.

File: src/content/guides/MYSQL/BROWSER/BROWSER.MD [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+2. Gib nun die URL `10.16.1.6` ein 
+
+   ![Screenshot 1](content/guides/MYSQL/BROWSER/screen1.png)
+
+3. Gib jetzt Benutzernamen und Passwort ein und drücke auf `OK`
+
+   ![Screenshot 2](content/guides/MYSQL/BROWSER/screen2.png)
+
+4. Nun kann wie gewohnt mit der Datenbank gearbeitet werden
+
```
Comment (Lordnoname) [inline src/content/guides/MYSQL/BROWSER/BROWSER.MD]: Please remove that line

File: src/content/guides/MYSQL/TERMINAL/TERMINAL.MD [hunk 2]
```diff
@@ -0,0 +1,11 @@
+1. Öffne das Terminal mit der Tastenkombination `Strg + Alt + T`
+
+2. Gib nun den Befehl `mysql -h 10.16.1.6 -u [Benutzername] -p` ein
+
+   ![Screenshot 1](content/guides/MYSQL/TERMINAL/screen1.png)
+
+3. Jetzt muss das Passwort eingegeben werden
```
Comment (Lordnoname) [inline src/content/guides/MYSQL/TERMINAL/TERMINAL.MD]: Please provide active sentences

**Comment by gilgda** [comment]
Got a LGTM from @DevVersion

**Comment by gilgda** [comment]
Fixed with  SHA 97ab053ff56fec33a76cfda079d12da68e8ffd48

**Pull request closed without merging**